.github/workflows/pr.yaml

```diff
@@ -38,3 +38,18 @@ jobs:
         uses: super-linter/super-linter/slim@v8.3.2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build:
+    name: Build all packages
```

Comment:
Suggestion, not required: Maybe name it Build all Bazel pakages so it doesn t confuse the reader if these are Bazel packages or packages uploaded to ghcr